base/system/diskpart/lang/pl-PL.rc

```diff
@@ -32,10 +32,10 @@ BEGIN
     IDS_DETAIL_INFO_HIBER_FILE_DSK "Dysk pliku hibernacji"
     IDS_DETAIL_INFO_CRASH_DSK "Dysk zrzutu awarii systemu"
     IDS_DETAIL_INFO_CLST_DSK "Dysk klastrowany"
-    IDS_DETAIL_PARTITION_NUMBER "Partition %lu\n"
-    IDS_DETAIL_PARTITION_TYPE "Type          : %02x\n"
-    IDS_DETAIL_PARTITION_HIDDEN "Hidden        : %s\n"
-    IDS_DETAIL_PARTITION_ACTIVE "Active        : %s\n"
+    IDS_DETAIL_PARTITION_NUMBER "Partycja %lu\n"
+    IDS_DETAIL_PARTITION_TYPE "Typ          : %02x\n"
+    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
+    IDS_DETAIL_PARTITION_ACTIVE "Aktywna        : %s\n"
```

Comment:
initial english:
    IDS_DETAIL_PARTITION_TYPE "Type          : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Hidden        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Active        : %s\n"

adding spaces before the opening " makes that easier to see for the human eye,
that your current:

    IDS_DETAIL_PARTITION_TYPE   "Typ          : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Aktywna        : %s\n"
    IDS_DETAIL_PARTITION_OFFSET "Offset in Byte: %I64u\n"

should become:

    IDS_DETAIL_PARTITION_TYPE   "Typ           : %02x\n"
    IDS_DETAIL_PARTITION_HIDDEN "Ukryta        : %s\n"
    IDS_DETAIL_PARTITION_ACTIVE "Aktywna       : %s\n"
    IDS_DETAIL_PARTITION_OFFSET "Offset in Byte: %I64u\n"

Reply:
it should/must be translated ;)...
You want list of volume in diskpart/cmd in ENGLISH ?😂🤣🤣🤣

Reply:
fixed